lib/momento/set_response.rb

```diff
@@ -9,9 +9,24 @@ def success?
 
     # The item was set.
     class Success < SetResponse
+      attr_accessor :key, :value
+
+      # rubocop:disable Lint/MissingSuper
+      def initialize(key:, value:)
```

Comment:
not sure whether we are currying key/value in this response in the other SDKs but I think I like it, so 👍

Reply:
PHP does it. https://github.com/momentohq/client-sdk-php/blob/main/src/Cache/CacheOperationTypes/CacheOperationTypes.php#L293